CHANGELOG.md

```diff
@@ -3,6 +3,120 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.6.0
+
+_Mar 3, 2022_
+
+A big thanks to the 15 contributors who made this release possible. Here are some highlights ✨:
+
+- 📦 Use the bundling scripts as the packages published by the [https://github.com/mui/material-ui](material-ui) repository (#3965) @flaviendelangle
+
+  The code structure and the bundling strategy have been modified to provide better isolation between components.
+  This brings tree shaking, allowing to reduce the bundle size. 🏋
+  Such modification is always a bit risky.
+  If you encounter problems with your build, please open an issue.
+  These issues will have high priority.
+
+- 🧼 Clean and document the columns selector (#4010) @flaviendelangle
+
+  To improve clarity, columns selectors have been renamed.
```

Comment:
Let's put the most important point right at the start of the sentence.

```suggestion
Columns selectors have been renamed to improve clarity.
```

Reply:
I don't know if it was a typo in the suggestion but if the line is part of the list item we need to keep the leading two spaces:

```suggestion
  Columns selectors have been renamed to improve clarity.
```

Reply:
My mistake there!